src/Magnum/Platform/GlfwApplication.h

```diff
@@ -436,6 +437,33 @@ class GlfwApplication {
         /** @copydoc Sdl2Application::redraw() */
         void redraw() { _flags |= Flag::Redraw; }
 
+    public:
+        enum class Cursor : UnsignedInt {
+            Arrow,      /**< Arrow */
+            IBeam,      /**< I-Beam */
+            Crosshair,  /**< Crosshair */
+            ResizeWE,   /**< Double arrow pointing west and east */
+            ResizeNS,   /**< Double arrow pointing north and south */
+            Hand        /**< Hand */
+        };
+
+    public:
+        /**
+         * @brief Create a cursor that can then be used
+         * 
+         * @see @ref setCursor()
+         */
+        void createCursor(Cursor cursor);
```

Comment:
What about having this function contained directly in `setCursor()` -- allocating a new cursor, if it's not there yet? I think that would be more user-friendly. Then you also wouldn't need anything in the `Configuration`.